infrastructure/resource_group_level/modules/clamav-aca.bicep

```diff
@@ -0,0 +1,88 @@
+param location string = resourceGroup().location
+param environment string
+param containerName string
+param acaSubnetId string
+param logAnalyticsWorkspaceName string
+
+var managedEnvironmentName = 'tfs-${environment}-clamav-env'
+var applicationInsightsName = 'tfs-${environment}-clamav-ai'
+
+resource workspace 'Microsoft.OperationalInsights/workspaces@2022-10-01' existing = {
+  name: logAnalyticsWorkspaceName
+}
+
+resource managedEnvironment 'Microsoft.App/managedEnvironments@2023-05-02-preview' = {
+  name: managedEnvironmentName
+  location: location
+  properties: {
+    vnetConfiguration: {
+      internal: true
+      infrastructureSubnetId: acaSubnetId
+    }
+    workloadProfiles: [
+      {
+        workloadProfileType: 'Consumption'
+        name: 'Consumption'
+      }
+    ]
+    appLogsConfiguration: {
+      destination: 'log-analytics'
+      logAnalyticsConfiguration: {
```

Comment:
I stuck log analytics (here) and Application Insights (below) in, as it was very easy to add.